.code-samples.meilisearch.yaml

```diff
@@ -848,3 +856,34 @@ tenant_token_guide_search_sdk_1: |-
     APIKey: token,
   })
   frontEndClient.Index("patient_medical_records").Search("blood test", &meilisearch.SearchRequest{});
+synonyms_guide_1: |-
+  synonyms := map[string][]string{
+    "great":      []string{"fantastic"},
+    "fantastic":  []string{"great"},
+  }
+  client.Index("movies").UpdateSynonyms(&synonyms)
+date_guide_Index_1: |-
+  jsonFile, _ := os.Open("games.json")
```

Comment:
I found it very "strange" to load a file in the code-samples, not just here but all the other SDKs I feel that it is wrong to do it and is quite "ugly"...

CC: @bidoubiwa

Reply:
For me, it's ok because it's only in certain places in the sample codes, and the doc team will surely be able to enlighten us on why in those specific places
Also, it's a quick and easy way to be able to generate/index documents quickly and test `Meilisearch` with the json documents available via the documentation, so for people who just want to try quickly how works `Meilisearch` I find it great!
Even if yes is quite "ugly" ...